### PR TITLE
[Ready for review] Pre book dash comms

### PR DESCRIPTION
**Pull request by pherterich**
### Summary

Info to be sent out to attendees before the book dash as laid out in #399 #397 #398 

### List of changes proposed in this PR (pull-request)

Initial drafts for both emails

### What should a reviewer concentrate their feedback on?

- [ ] Any important info missing or wrong?
- [ ] Rephrase here and there please
- [ ] Fix typos

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).

### CODE REVIEW

**Comment by rainsworth** [comment]
I've added a line on the CoC, we could also potentially promote the Gitter Channel? Not sure how much information is too much information at this stage - as ways of getting in touch is also already listed in the contributing guidelines which we have linked to.

**Comment by pherterich** [comment]
I can add a line on that - are we missing anything else or waiting for further info or can the Manchester mail go out later today you think?

**Comment by rainsworth** [comment]
Manchester should be able to go out today - although I haven't had official confirmation on Jamie's Italian.

**Comment by pherterich** [comment]
Not sure where my afternoon went, will schedule Manchester emails this weekend to go out on Monday morning. Don't want to encourage working at the weekend, but if you have any crazy good ideas on Sunday morning, there is still time to add them in.

**Comment by pherterich** [comment]
Note to self: rewrite last sentence as the booking people seem to send tickets to attendees directly...

File: workshops/book-dash/before_workshop.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+Talks should be 3 minutes long and be without the support of slides as the venue will not be able to provide any equipment to display them.
+Please let us know on arrival if you would like to give an elevator pitch.
+
+## On the day
+
+We are very excited that an illustrator from [scriberia](http://www.scriberia.co.uk/) will join us to document the sprint and provide some amazing visualisations for the handbook.
+
+### Getting to the venue
+
+We will start the book dash before the British Library officially opens at 9:30am so you will need to enter through Gate 8 of The British Library, the staff entrance on Midland Road opposite (one of the exits from) St Pancras station ([here are instructions on how to get to The Alan Turing Institute](https://www.turing.ac.uk/about-us/how-get-alan-turing-institute)).
```
Comment (KirstieJane) [inline workshops/book-dash/before_workshop.md]: Can you change this link to https://whitakerlab.github.io/visit? It has a nice picture of the gate which I think makes it easier for folks to follow.

**Comment by pherterich** [comment]
Merging this as the last email just went out

**Pull request merged**